content/content/oop_macro.md

```diff
@@ -55,7 +55,7 @@ macro class*(head, body: untyped): untyped =
     # --------------------
     # Ident !"Animal"
     typeName = head
-    baseName = ident("RootObj")
+    baseName = newIdentNode("RootObj")
```

Comment:
Only because I was keeping to versions of the file. 
However I was very confused with ident, cause there two of them (actually three):
proc ident(n: NimNode): NimIdent {..}
proc ident(name: string): NimNode {..}
and the last one, used here, is just an alias for newIdentNode...